components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,60 @@
+@import "@hods/sass-base";
+
+.hods-timeline {
+  overflow: hidden;
+  position: relative;
+
+  &__item {
+    @include govuk-responsive-padding(2, "bottom");
+    padding-left: 22px;
+    position: relative;
+
+    &:hover {
+      background-color: govuk-colour("mid-grey");
+    }
+
+    &:before { /* styling for lines in margin */
+      background-color: govuk-colour("blue");
+      content: "";
+      @include govuk-responsive-padding(1, "top");
+      @include govuk-responsive-padding(3, "right");
+      left: 0;
+      position: absolute;
+      @include govuk-responsive-margin(1, "top");
+      z-index: 1;
+    }
+  }
+
+  &:before { /* styling for lines in margin */
+    background-color: govuk-colour("blue");
+    content: "";
+    height: 100%;
+    left: 0;
+    position: absolute;
+    margin-top: 8px;
+    @include govuk-responsive-padding(1, "left");
+    z-index: 1;
+  }
+
+  &__title {
+    @include govuk-font(16, 'bold');
+    display: inline;
+  }
+
+  &__by {
+    @include govuk-font(16);
+    color: govuk-colour('dark-grey');
+    display: inline;
+  }
+
+  &__date {
+    @include govuk-font(14);
```

Comment:
I think this should be at least `16`. (We wouldn't normally go as low as `14` for accessibility reasons.)

Reply:
Have changed this now